operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -413,6 +409,42 @@ spec:
 # Only used when cache_enabled is true.
 #    ---
 #    cache_duration: 300000000
+#
+# Kiali can cache VirtualService,DestinationRule,Gateway and ServiceEntry Istio resources if they are present
+# on this list of Istio types. Other Istio types are not yet supported.
+#    ---
+#    cache_istio_types:
+#    - "DestinationRule"
+#    - "Gateway"
+#    - "ServiceEntry"
+#    - "VirtualService"
+#
+# List of namespaces or regex defining namespaces to include in a cache.
+#    ---
+#    cache_namespaces: []
+#
+# Cache duration expressed in nanoseconds
```

Comment:
As it's configuration, shouldn't it be something more human-friendly like seconds? I understand we're targeting short-lived cache, but to the point of nanos? :-)

Reply:
Yeah, good point, I used the native field of the API which is in nanos, that's why I propagated.